chameleon.py

```diff
@@ -68,20 +90,28 @@ def stop():
 def _get_cherrypy_page_handler(status_code, content_type, content):
     class Foo(object):
         @cherrypy.expose
+        @cherrypy.tools.json_out()
```

Comment:
Yeah it fixed a Unicode problem. 

https://bitbucket.org/cherrypy/cherrypy/src/92e4579f029f37ecfff833a71d359a8e914155f3/cherrypy/_cprequest.py?at=default#cl-802

Let me look into what that decorator does. If it's true it only enables json based on content type it may conform to the agnostic design.